### ISSUE TITLE
The "back" button in the Crane app does not work with screen readers

### ISSUE THREAD

**Issue by vick08** [flutter/flutter#82553]
Describe your issue and include the command you're running, flutter_web version, browser version

Steps to reproduce:
1. Launch VoiceOver screen reader on Mac OS (Command + POWER button three times or ask Siri).
2. Using Chrome or Safari go to gallery.flutter.dev, tab to the "Crane" link and press the ENTER key to go inside the app.
3. Tab around between the widgets on the page, then shift+tab to the BACK button and observe.

Expected results:
1. The shift+tab should be able to reach and focus on the BACK button.
2. The screen reader should announce the element in focus.

Observed results:
None of the above happens.
